### PR TITLE
added copy to clipboard and mailto client buttons

### PR DESCRIPTION
**Pull request by gurjit03**
Added 2 buttons copy to clipboard and mailto client. Both the buttons actions are also working

### CODE REVIEW

**Comment by gurjit03** [comment]
Also forget to mention, moved the styles to a seperate file

File: src/Tooltip.js [hunk 1]
```diff
@@ -1,64 +1,39 @@
...
+      <CopyToClipboard text={emailAddress} onCopy={() => alert('copied.')}>
+        <button className={className + '__tooltip--copy-email-address __tooltip-body-btn'} style={toolTipBodyBtnStyles}>
+          Copy Email Address To ClipBoard
+        </button>
+      </CopyToClipboard>
+
+      <button
+        className={className + '__tooltip--compose-default-email __tooltip-body-btn'}
+        style={toolTipBodyBtnStyles}
+        onClick={() => openEmailClient(emailAddress)}
```
Comment (alexpricedev) [inline src/Tooltip.js]: Do you think we need to make this a `button` with and `onClick`?  We could just make it an `<a href="">`??

Comment (gurjit03) [reply src/Tooltip.js]: was being consitent here, as both looks more like a button when look in the design as well. 
Any trade off with using button over anchor here ?

Comment (alexpricedev) [reply src/Tooltip.js]: Semantics are more important than how the button looks when rendered IMO. If we use a `<a>` we can simply write: `<a href={`mailto:${emailAddress}`}>` and be done with it. No need to click handlers etc as the browser handles `mailto:` prefixes

Comment (gurjit03) [reply src/Tooltip.js]: @alexpriceonline I have changed the button to the `<a>`. Can you verify this commit :)

**Review by alexpricedev — APPROVED** [review]
🚀

**Pull request merged**